test/integration_tests.jl

```diff
@@ -16,5 +16,5 @@ end
 @testset "matmul example" begin
     @assert (@ballocated f_matmul()) === 18_304
     val, record = record_allocations(f_matmul)
-    @test (@ballocated avoid_allocations($record, f_matmul)) == 352
+    @test (@ballocated avoid_allocations($record, f_matmul)) == 256
```

Comment:
This line is failing on CI says 320 allocations?

Reply:
errr... I got 256 on my machine however. 

```julia
julia> versioninfo()
Julia Version 1.3.0
Commit 46ce4d7933 (2019-11-26 06:09 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin19.0.0)
  CPU: Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
```

what about changing it to <=252 then?